rllib/policy/tf_policy_template.py

```diff
@@ -48,55 +85,75 @@ def build_tf_policy(name,
     otherwise they will fail in eager mode execution. Variable should only
     be created in make_model (if defined).
 
-    Arguments:
-        name (str): name of the policy (e.g., "PPOTFPolicy")
-        loss_fn (func): function that returns a loss tensor as arguments
-            (policy, model, dist_class, train_batch)
-        get_default_config (func): optional function that returns the default
-            config to merge with any overrides
-        postprocess_fn (func): optional experience postprocessing function
-            that takes the same args as Policy.postprocess_trajectory()
-        stats_fn (func): optional function that returns a dict of
-            TF fetches given the policy and batch input tensors
-        optimizer_fn (func): optional function that returns a tf.Optimizer
-            given the policy and config
-        gradients_fn (func): optional function that returns a list of gradients
-            given (policy, optimizer, loss). If not specified, this
-            defaults to optimizer.compute_gradients(loss)
-        apply_gradients_fn (func): optional function that returns an apply
-            gradients op given (policy, optimizer, grads_and_vars)
-        grad_stats_fn (func): optional function that returns a dict of
-            TF fetches given the policy, batch input, and gradient tensors
-        extra_action_fetches_fn (func): optional function that returns
-            a dict of TF fetches given the policy object
-        extra_learn_fetches_fn (func): optional function that returns a dict of
-            extra values to fetch and return when learning on a batch
-        validate_spaces (Optional[callable]): Optional callable that takes the
-            Policy, observation_space, action_space, and config to check for
-            correctness.
-        before_init (func): optional function to run at the beginning of
-            policy init that takes the same arguments as the policy constructor
-        before_loss_init (func): optional function to run prior to loss
-            init that takes the same arguments as the policy constructor
-        after_init (func): optional function to run at the end of policy init
-            that takes the same arguments as the policy constructor
-        make_model (func): optional function that returns a ModelV2 object
-            given (policy, obs_space, action_space, config).
+    Args:
+        name (str): Name of the policy (e.g., "PPOTFPolicy").
+        loss_fn (Callable[[Policy, ModelV2, type, SampleBatch], TensorType]):
+            Callable for calculating a loss tensor.
+        get_default_config (Optional[Callable[[None], TrainerConfigDict]]):
```

Comment:
It is good to add comments regarding what these functions default to if none. Like if get_default_config is none, it will get it stuff from trainer.py.

Reply:
Great point! Will add this. :)